charts/atlantis/templates/secret-netrc.yaml

```diff
@@ -0,0 +1,10 @@
+{{- if .Values.netrc}}
```

Comment:
Instead of using a .netrc file, couldn't you use this flag as an env var `--write-git-creds` ?

https://www.runatlantis.io/docs/server-configuration.html#write-git-creds

Reply:
Let me look into it, but for my exact use case we need to pull the modules from Artifactory. So technically git creds don't come into play here.